cbor/src/main/java/com/fasterxml/jackson/dataformat/cbor/CBORParser.java

```diff
@@ -245,6 +322,35 @@ private Feature(boolean defaultState) {
      */
     protected int _typeByte;
 
+    /**
+     * The depth of nested objects.
+     */
+    protected int _nestedDepth = 0;
+
+    /**
+     * Type to keep track of a list of string references. A depth is stored to know when to pop the
+     * references off the stack for nested namespaces.
+     */
```

Comment:
Add `@since 2.15` for new classes, public methods in public API.

Reply:
I did a quick check and added it to this and the new tag constants in CBORConstants. Let me know if I missed any others.

Reply:
Thanks. I think it's fine; if some are missing I'll add a note.